README.md

```diff
@@ -102,12 +102,13 @@ For front-end applications and node <v4, please use `require('fluture/es5')`.
 [<img src="https://raw.github.com/fantasyland/fantasy-land/master/logo.png" align="right" width="82" height="82" alt="Fantasy Land" />][FL]
 [<img src="https://raw.githubusercontent.com/rpominov/static-land/master/logo/logo.png" align="right" height="82" alt="Static Land" />][6]
 
-Fluture implements [FantasyLand 1][FL1], [FantasyLand 2][FL2],
-[FantasyLand 3][FL3] and [Static Land][6] compatible `Functor`, `Bifunctor`,
-`Apply`, `Applicative`, `Chain`, `ChainRec` and `Monad`.
-
-FantasyLand 0.x is *mostly* supported. Everything but `Apply` (`ap`) is, this
-means that dispatchers to FantasyLand 0.x `ap` will not work.
+* `Future` implements [FantasyLand 1][FL1], [FantasyLand 2][FL2],
+  [FantasyLand 3][FL3] and [Static Land][6] compatible `Bifunctor`, `Monad` and
+  `ChainRec` (`of`, `ap`, `map`, `bimap`, `chain`, `chainRec`). FantasyLand 0.x
+  is *mostly* supported. Everything but `Apply` (`ap`) is, this means that
+  dispatchers to FantasyLand 0.x `ap` will not work.
+* `Future.Par` implements [FantasyLand 3][FL3] `Alternative` (`of`, `zero`, `map`, `ap`, `alt`).
+* Instances can be identified by, and are compared using, [Sanctuary Type Identifiers][STI].
```

Comment:
What does it mean to "identify" a future?

What is meant by "compared"? Are you saying that a future is not considered equal to a non-future because the type identifiers do not match?